app/assets/components/tagify/.bower.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Whilst not important from a functionality point of view, I believe all external assets should be placed in vendor/assets with the rails asset pipeline ?